.env.local.example

```diff
@@ -1,3 +0,0 @@
-# Copy this file and rename it to `.env.local` to configure your own WCL API key. You can get your own API key from the bottom of the Warcraft Logs settings: https://www.warcraftlogs.com/profile
```

Comment:
please don't delete this file as its used as an example

Reply:
Forgot I didn't make a copy just renamed so added the file back